app/helpers/duration_formatter_helper.rb

```diff
@@ -0,0 +1,6 @@
+module DurationFormatterHelper
```

Comment:
Could this go in the MetricsFormatterHelper module?

Reply:
Done c13e348